webpack/localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -198,10 +198,7 @@ export class LocalizationPlugin implements Webpack.Plugin {
 
     if (isWebpackDevServer) {
       if (typingsPreprocessor) {
-        compiler.hooks.afterEnvironment.tapPromise(
-          PLUGIN_NAME,
-          async () => await typingsPreprocessor!.runWatcherAsync()
-        );
+        compiler.hooks.afterEnvironment.tap(PLUGIN_NAME, () => typingsPreprocessor!.runWatcherAsync());
```

Comment:
Is this no longer async, or do we just not need to wait for the promise resolution?

Reply:
`afterEnvironment` is apparently sync (not sure how this worked before honestly), so here we're just firing-and-forgetting `runWatcherAsync`.